pkg/services/pluginsintegration/plugintest/plugins_test.go

```diff
@@ -49,11 +56,92 @@ func TestMain(m *testing.M) {
 	testsuite.Run(m)
 }
 
+// This test should run before TestIntegrationPluginManager because this test relies on having a pre-existing Admin user
+// and because the SQLStore instance is shared between tests, this test does all the necessary setup
+func TestIntegrationPluginDashboards(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	provisioningFile := filepath.Join(t.TempDir(), "apps.yaml")
+	err := os.WriteFile(provisioningFile, []byte(`apiVersion: 1
+
+apps:
+  - type: test-app
+    org_id: 1
+    org_name: Main Org.
+    disabled: false
+    jsonData:
+      apiKey: "test-api-key"
+    secureJsonData:
+      secretKey: "test-secret-key"`), 0644)
```

Comment:
What is the reason why this isn't another file under testdata but created on the "fly"?

Reply:
Great point - have fixed!